extensions/npm/main.ts

```diff
@@ -171,21 +176,46 @@ if (!output.success) {
 async function checkDryRun(subcommand: string, args: string[]) {
   console.log(`[${green("phylum")}] Updating lockfile…`);
 
-  const cmd = new Deno.Command("npm", {
-    args: [
-      subcommand,
-      "--package-lock-only",
-      "--ignore-scripts",
-      ...args,
-    ],
-    stdout: "inherit",
-    stderr: "inherit",
-    stdin: "inherit",
-  });
-  const status = await cmd.spawn().status;
+  // Ensure lockfile is up to date.
+  let status;
+  if (
+    "install".startsWith(subcommand) ||
+    "isntall".startsWith(subcommand) ||
+    "add".startsWith(subcommand)
+  ) {
```

Comment:
Great question, initially I didn't add it because it doesn't have a `--package-lock-only` flag. But even without it it should still be better since we otherwise don't check new packages.

This isn't ideal, because malicious new updates will still be installed, but I don't think there's much we can do?

Reply:
> initially I didn't add it because it doesn't have a `--package-lock-only` flag.

`package-lock-only` [explicitly mentions `update`][1] as one of the commands that it affects... So I don't think this is correct.

[1]: https://docs.npmjs.com/cli/v7/using-npm/config#package-lock-only

Reply:
You're right, I confirmed that it does indeed work.